### DISCUSSION TITLE
Welcome to stable-diffusion-ps-pea Discussions!

### DISCUSSION THREAD

**Discussion by huchenlei** [huchenlei/stable-diffusion-ps-pea#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by camoody1** [comment]
@huchenlei 

Hi! I'm so glad you're building a great plugin for Photopea. I struggle a bit, though, being able to make simple outpainting to extend my canvas. There are so many settings, that I'm really not sure how best to get things done simply.

I just saw a post on Reddit for a plugin someone has built for Krita and it looks about as easy to use as Photoshop's Generative Fill. 
https://reddit.com/r/StableDiffusion/s/jGWdTHo9b4

Do you think there's any way you could add a "Quick Fill" option for users who don't fully understand all your other switches and settings? Or perhaps record a tutorial video showing how to do some simple outpainting and Inpainting using your plugin?

Thank you
